### PR TITLE
DL: Enable transfer learning for multi model training

### PR DESCRIPTION
**Pull request by orhankislal**
JIRA: MADLIB-1389

This commit enables transfer learning for multi-model training by
copying the weights from the model arch table.

- [x] Add the module name, JIRA# to PR/commit and description.
- [x] Add tests for the change.

### CODE REVIEW

**Comment by orhankislal** [comment]
jenkins please retest this

**Comment by khannaekta** [comment]
retest this please

**Comment by fmcquillan99** [comment]
(1)
Train result, validation loss per iteration:
```
[0.733925223350525, 0.501948714256287, 0.401886612176895, 0.494124263525009, 0.384743869304657, 0.300327688455582, 0.368834614753723, 0.319071084260941, 0.191268876194954, 0.173252046108246]
```

After transfer learning, seems like the loss on subsequent training picks up from where this ^^^ leaves off:
```
[0.16432112455368, 0.164827600121498, 0.15946963429451, 0.156749069690704, 0.144549146294594, 0.142323225736618, 0.138157039880753, 0.127701371908188, 0.127511203289032, 0.12169610708952]

[0.151703342795372, 0.15974634885788, 0.0791483223438263, 0.0740548744797707, 0.0698189586400986, 0.0977403670549393, 0.0737137421965599, 0.0555036813020706, 0.0894732773303986, 0.102973714470863]

[0.14056433737278, 0.115160055458546, 0.105597652494907, 0.0925526767969131, 0.0991481989622116, 0.0907492190599442, 0.10708586871624, 0.142711892724037, 0.0837283283472061, 0.0745012909173965]
etc
```
